etc/profile-m-z/mullvad-browser.profile

```diff
@@ -0,0 +1,76 @@
+# Firejail profile for mullvad-browser
+# Description: Privacy-focused web browser developed in a collaboration between Mullvad VPN and the Tor Project
+# This file is overwritten after every install/update
+# Persistent local customizations
+include mullvad-browser.local
+# Persistent global definitions
+include globals.local
+
+# If you installed under ${HOME}, put the below lines in your mullvad-browser.local
+#ignore noexec ${HOME}
+#noblacklist ${HOME}/.local/share/mullvad-browser
+
+noblacklist ${HOME}/.config/mullvad-browser-flags.conf
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python2.inc
+include allow-python3.inc
+
+blacklist /srv
+blacklist /sys/class/net
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+mkfile ${HOME}/.config/mullvad-browser-flags.conf
+# If you installed under ${HOME}, put the below lines in your mullvad-browser.local
+#mkdir ${HOME}/.local/share/mullvad-browser
+#whitelist ${HOME}/.local/share/mullvad-browser
```

Comment:
Can be moved below `#noblacklist ${HOME}/.local/share/mullvad-browser` to avoid
duplicated comments.

Reply:
Fixed, thanks!